var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -57,10 +58,6 @@ class NetcdfFortran(AutotoolsPackage):
     # https://github.com/Unidata/netcdf-fortran/commit/0a11f580faebbc1c4dce68bf5135709d1c7c7cc1#diff-67e997bcfdac55191033d57a16d1408a
     patch('excessive_linking.patch', when='@4.4.5')
 
-    # Parallel builds do not work in the fortran directory. This patch is
-    # derived from https://github.com/Unidata/netcdf-fortran/pull/211
-    patch('no_parallel_build.patch', when='@4.5.2')
-
```

Comment:
@haampie are you sure we don't need the patch when installing version `4.5.2`?

Reply:
You will need it, but I don't see the point of "backporting" patches from 4.5.3 to 4.5.2. Just build `@:4.5` if you need a stable 4.5.x release, and if you insist on building an old patch release, you're volunteering to have bugs :man_shrugging:.

Reply:
So we drop support for a version without deprecating it?

Reply:
Let me just undo this change so that we can get the new version merged.

My reasoning: I assumed somebody added the patch because 4.5.3 was not yet released, now that 4.5.3 is released, the patch is no longer necessary. Otherwise what do we have, version 4.5.2-and-a-half?